lib/src/services/git_tools.dart

```diff
@@ -119,6 +123,32 @@ class GitTools {
       throw FvmInternalError('Could not git clone $version');
     }
 
+    if (isCommit) {
```

Comment:
@ened Could we move this into a separe method in git tools, and call if it's a commit?